elks/elks-small.ld

```diff
@@ -11,7 +11,7 @@ SECTIONS {
 		LONG(SIZEOF (.text));
 		LONG(SIZEOF (.data));
 		LONG (SIZEOF (.bss));
-		LONG (entry);    /* entry point */
```

Comment:
I intentionally renamed the entry point symbol when moving from BCC to GCC, to detect and review all that entries, plus to forbid any try to link from some old BCC stuff... what is the reason to rename it back today ?

Reply:
Hello @mfld-fr,

I see.  My initial reasoning was that many other environments, including GCC for Linux, and also `gcc-ia16` for MS-DOS, use `_start` as the entry point.  And --- I think this is quite a good reason --- `entry` is not a reserved identifier in ANSI C, and thus may clash with valid C code.

I think there is a low risk of accidentally linking with stuff meant for `bcc` --- the `gcc-ia16` toolchain does not prepend underscores to C symbols (by default), while `bcc` does.

Thank you!

Reply:
OK then if no accidental linking. Thanks !